src/main/java/com/ngs/stash/externalhooks/hook/ExternalMergeCheckHook.java

```diff
@@ -255,6 +267,7 @@ public RepositoryHookResult preUpdate(PreRepositoryHookContext context, PullRequ
         final String summaryMsg = wasAccepted ? "External hooks check was successful." : "External hooks check failed.";
 
         check.setWasAccepted(wasAccepted);
+        check.setLastExceptionSummary(String.format("%s\n%s", summaryMsg, detailMsg));
```

Comment:
Result of this set will be overridden by next line, maybe there you need to set it conditionally depending on value of `tooltip_results`?

Reply:
I think this was mistakenly left in during my development. I had meant to keep the line that follows.